### PR TITLE
fix(workitem status icon): added support for icon color

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Added support for color of work item status icons in ngx-widgets

### CODE REVIEW

**Comment by fabric8cd** [comment]
@divyanshiGupta fabric8/fabric8-ui:SNAPSHOT-PR-2187-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2187-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@divyanshiGupta fabric8/fabric8-ui:SNAPSHOT-PR-2187-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2187-fabric8-planner.badger.fabric8.io

**Comment by debloper** [comment]
You need to rebase your patch; it looks messed up with #2121.
Once rebased, you should force push only the new commits.
Take help from someone in office if this seems confusing.

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
   <!-- info area -->
   <div class="list-view-pf-main-info">
     <div class="list-view-pf-left type f8-wi__list-witype">
-      <span almIcon [iconType]="workItem.attributes['system.state']" class="color-grey pull-left"></span>
+      <span  class="pull-left" almIcon [iconType]="workItem.attributes['system.state']"></span>
```
Comment (debloper) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: extra space before `class`
also, if you're just dropping `color-grey`, then no need to change the attribute ordering of `almIcon` and `class`

Comment (divyanshiGupta) [reply src/app/components/work-item-list-entry/work-item-list-entry.component.html]: Got it

**Comment by fabric8cd** [comment]
@divyanshiGupta fabric8/fabric8-ui:SNAPSHOT-PR-2187-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2187-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@divyanshiGupta fabric8/fabric8-ui:SNAPSHOT-PR-2187-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2187-fabric8-planner.badger.fabric8.io

**Comment by divyanshiGupta** [comment]
Closing this PR as I have a raised a new PR with the same changes as this and some new as well.
Refer : #2288

**Pull request closed without merging**